README.md

```diff
@@ -96,4 +96,4 @@ The `travis.py` script try to be clever:
 
 It is possible to build local image for testing with the following command:
 
-    CI_HELPER_VERSION=0.0.5 MODD_VERSION=0.5 JAVA_VERSION=10.0.1-jdk-slim VERSION=10 python travis.py --language java --pull-request=true
+    CI_HELPER_VERSION=0.0.6 MODD_VERSION=0.5 JAVA_VERSION=jdk-11.0.3_7-debian-slim VERSION=10 python travis.py --language java --pull-request=true
```

Comment:
The command line is also missing `JAVA_MAJOR_VERSION`, and `VERSION=10` should be replaced by `VERSION=11`.

Reply:
Yeah I forgot to update `VERSION`. :+1:

`JAVA_MAJOR_VERSION` is in fact declared in `travis.py` and initlialized from `VERSION` (I didn't add it, it was already done).

Reply:
I'll clean it a bit too, we need it for the sed (`Dockerfile.tpl` to `Dockerfile`), but not for the Docker env anymore (it was used to do JDK 8 specific tasks)

Reply:
The example has been fixed